nixio/test/test_multi_tag.py

```diff
@@ -102,6 +102,19 @@ def tearDown(self):
         self.file.close()
         self.tmpdir.cleanup()
 
+    def test_multi_tag_new_constructor(self):
```

Comment:
Test should also check if DataArrays were created with the expected names.

Reply:
More importantly, check that there's two DataArrays with the expected names on the appropriate Block and add a check that those data arrays have the same ID as the position and extent references.